packages/rocketchat-integrations/server/lib/validation.js

```diff
@@ -0,0 +1,155 @@
+/* global Babel */
+const scopedChannels = ['all_public_channels', 'all_private_groups', 'all_direct_messages'];
+const validChannelChars = ['@', '#'];
+
+function _verifyRequiredFields(integration) {
+	if (!integration.event || !Match.test(integration.event, String) || integration.event.trim() === '' || !RocketChat.integrations.outgoingEvents[integration.event]) {
+		throw new Meteor.Error('error-invalid-event-type', 'Invalid event type', { function: 'validateOutgoing' });
```

Comment:
Honestly not sure, as these are internal ones and the "bigger" function that is being used is actually `validateOutgoing`.